gradle.properties

```diff
@@ -1,5 +1,4 @@
 group=com.marklogic
-version=5.5-SNAPSHOT
-releaseVersion=5.5.3
```

Comment:
@georgeajit  @SameeraPriyathamTadikonda  Any idea what `releaseVersion` may have been used for? There are zero references to it in the codebase. And to my knowledge, `version` should be the only thing required for publishing an artifact.